backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -135,6 +146,20 @@ export const executeTransaction = async (
 
     await queryRunner.commitTransaction()
     logger.info(`commit Transaction successful...`)
+
+    const eventTransactionSend = new EventTransactionSend()
+    eventTransactionSend.userId = transactionSend.userId
+    eventTransactionSend.xUserId = transactionSend.linkedUserId
+    eventTransactionSend.transactionId = transactionSend.id
+    eventTransactionSend.amount = transactionSend.amount
```

Comment:
negative

Reply:
same

Reply:
As the amount in the transaction table for the send event is negative I suggest to have a negative value here as well.

Reply:
as before, sorry for that, clearly my mistake, solved